pkg/kubelet/apis/config/validation/validation.go

```diff
@@ -25,6 +25,8 @@ import (
 	utilvalidation "k8s.io/apimachinery/pkg/util/validation"
 	utilfeature "k8s.io/apiserver/pkg/util/feature"
 	"k8s.io/component-base/logs"
+	// As the JSON format is not yet GAed, an explicit registration is required
```

Comment:
Please put the line in ./cmd/kubelet/kubelet.go

Reply:
Updated. Thanks! 😄